src/utils/windowUtils.ts

```diff
@@ -0,0 +1,44 @@
+import { BrowserWindow } from '@electron/remote';
+import Electron from 'electron';
+
+export function getWindow(): Electron.BrowserWindow {
+  const [win] = BrowserWindow.getAllWindows();
+  return win;
+}
+
+export function getView(): Electron.BrowserView | null {
+  const win = getWindow();
+  return win.getBrowserView();
```

Comment:
Is win guaranteed, or could this error out?
Also why is | null the correct return type?

Reply:
Yes, `win` is guaranteed 👍